packages/components/src/SVGIcon/SVGIcon.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import SVGIcon from "./SVGIcon";
 import type { Story } from "@storybook/react";
 import Text from "../Text";
-import Tokens from "@chanzuckerberg/eds-tokens/json/variables.json";
+import Tokens from "@chanzuckerberg/eds-tokens/dist/json/variables.json";
```

Comment:
This is an implication of publishing the entire package (instead of just dist/), and is a result of https://github.com/lerna/lerna/issues/2631.

We'd need to go update anywhere this is used in Traject. I think it's fine, and simplifies the build process a little bit. 

But we could alternatively output the built files to `packages/tokens/` instead of `packages/tokens/dist`, since they are committed to the repo anyway.

---

Note that this same issue doesn't affect eds-components, since we can still tell npm that dist/index.js is the "main" file.